### PR TITLE
[ASDisplayNode] Expose default Texture-set accessibility values as properties

### PR DESCRIPTION
**Pull request by jiawernlim**
Expose default Texture-set accessibility values as properties in ASDisplayNode, and added relevant overrides and tests.

This allows clients to look up the default a11y values set by Texture in ASDisplayNode (and its subclasses) and reset them if needed.

### CODE REVIEW

File: Source/ASButtonNode.mm [hunk 1]
```diff
@@ -544,6 +540,16 @@ - (ASLayoutSpec *)layoutSpecThatFits:(ASSizeRange)constrainedSize
   return spec;
 }
 
+- (NSString *)defaultAccessibilityLabel {
```
Comment (maicki) [inline Source/ASButtonNode.mm]: Can you add a space newline at the end please.

File: Source/ASButtonNode.mm [hunk 2]
```diff
@@ -544,6 +540,16 @@ - (ASLayoutSpec *)layoutSpecThatFits:(ASSizeRange)constrainedSize
...
+  ASLockScopeSelf();
+  return _titleNode.accessibilityLabel;
+}
+
+- (UIAccessibilityTraits)defaultAccessibilityTraits {
```
Comment (maicki) [inline Source/ASButtonNode.mm]: Can you add a space newline at the end please.

File: Source/ASDisplayNode.mm [hunk 3]
```diff
@@ -3631,6 +3631,26 @@ - (BOOL)isAccessibilityContainer
   return _isAccessibilityContainer;
 }
 
+- (NSString *)defaultAccessibilityLabel {
```
Comment (maicki) [inline Source/ASDisplayNode.mm]: Please add a newline everywhere here.

File: Source/ASDisplayNode+Beta.h [hunk 4]
```diff
@@ -96,6 +96,17 @@ typedef struct {
  */
 @property BOOL isAccessibilityContainer;
 
+/**
+ * @abstract Returns the default accessibility property values set by Texture on this node. For
+ * example, the default accessibility label for a text node may be its text content, while most
+ * other nodes would have nil default labels.
+ */
+@property(nullable, readonly, copy) NSString *defaultAccessibilityLabel;
```
Comment (maicki) [inline Source/ASDisplayNode+Beta.h]: Please add a space after `@property`

File: Source/ASTextNode2.mm [hunk 5]
```diff
@@ -209,6 +209,15 @@ - (BOOL)supportsLayerBacking
   return YES;
 }
 
+- (NSString *)defaultAccessibilityLabel {
```
Comment (maicki) [inline Source/ASTextNode2.mm]: Please add a newline here and the next one too

Comment (maicki) [reply Source/ASTextNode2.mm]: Can you also add the `defaultAccessibilityLabel` to `ASTextNode` please.

Comment (jiawernlim) [reply Source/ASTextNode2.mm]: Done, added the default a11y stuff to ASTextNode.

**Pull request merged**